scripts/clear-all-fault-leds.sh

```diff
@@ -4,6 +4,14 @@
 # /xyz/openbmc_project/State/Decorator/OperationalStatusManager
 # to true or false.
 
+# Skip running this script during a warm reboot(graceful/force).
+current_chassis_status=$(busctl get-property xyz.openbmc_project.State.Chassis /xyz/openbmc_project/state/chassis0 xyz.openbmc_project.State.Chassis CurrentPowerState | cut -d" " -f2)
+
+if [ "${current_chassis_status}" == "\"xyz.openbmc_project.State.Chassis.PowerState.On\"" ]; then
+    echo "Current chassis power state is , $current_chassis_status . EXIT clear-all-fault-leds.sh SCRIPT SUCCESSFULLY WITHOUT DOING ANY FAULT LED RESET."
```

Comment:
Why is the script shouting? :smile: Please remove the all CAPS.

Reply:
I gave the EXIT statement in caps, why because

The obmc-clear-all-faults service file runs successfully and it throws this in the journal
May 09 09:47:48 ever10bmc systemd[1]: **Finished Clears all fault LED to true and remove critical associations.**

But actually the fault leds are not cleared. So to make the EXIT statement (from the script) more prominent I gave it in caps :)

Reply:
removed all CAPS